contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -103,6 +103,9 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   mapping(uint256 => bytes) internal s_upkeepOffchainConfig; // general config set by users for each upkeep
   mapping(uint256 => bytes) internal s_upkeepPrivilegeConfig; // general config set by an administrative role for an upkeep
   mapping(address => bytes) internal s_adminPrivilegeConfig; // general config set by an administrative role for an admin
+  // billing
+  mapping(address => BillingConfig) internal s_billingConfigs; // billing configurations for different tokens
+  address[] internal s_billingTokens; // list of billing tokens
```

Comment:
To quickly get the list of billing tokens, I have this address list. would like to hear if there is better solution.